app/views/users/_user.html.haml

```diff
@@ -15,7 +15,7 @@
         = h(@user.company) unless @user.company.blank?
       %br
       %div
-        #{t :email}: #{auto_link(@user.email)}
+        #{t :email}: #{auto_link(@user.alt_email)}
```

Comment:
Hi there, I'm not sure this is intended behaviour. We want to show the `email` and optionally the `alt_email`. Is there a problem you were trying to fix? Thanks.